src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/UserTransformer.kt

```diff
@@ -38,7 +40,19 @@ class UserTransformer(
     )
   }
   fun transformJpaToApi(jpa: UserEntity, serviceName: ServiceName) = when (serviceName) {
-    ServiceName.approvedPremises, ServiceName.cas2 -> ApprovedPremisesUser(
+    ServiceName.cas2 -> Cas2User(
```

Comment:
I think it might be simpler (as well as more meaningful) if you were to throw an exception if the `ServiceName` is "cas2" -- because a `Cas2User` of this form doesn't exist. CAS2 users are from the NOMIS/DPS side and so:

- don't have a Delius username
- don't have a probation region
- don't have qualifications